app/views/categories/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'user_index' %>
+<% end %>
+
+<% content_for :page_name, "New Category" %>
+
+<%= render 'helper/navbar' %>
+
+<div class="container-fluid">
+  <%= form_for @category, url: categories_path, method: :post do |f| %>
+    <div class="mt-3 mb-3">
+      <%= f.text_field :name, class: "form-control", placeholder: "Name" %>
+    </div>
+    <div class="mb-3">
+      <%= f.text_field :icon, class: "form-control", placeholder: "icon" %>
+    </div>
+    <button type="submit" class="btn btn-primary" style="width: 100%;">Create Category</button>
+  <% end %>
+</div>
```

Comment:
- According to the project requirements, there should be a back (<) button on this page that leads back to the home page, please fix this and make sure it is styled according to the design template 😄 

Design template:
![bajlntshaet](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/ec424e73-3600-4db6-a346-167dae2fae3f)

Reply:
done